README.md

```diff
@@ -1 +1,205 @@
+
+[Live demo](https://strong-travesseiro-0c6db9.netlify.app/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Presentation of the project -->
+
+## 🔭 Video Presentation <a name="future-features"></a>
+
+Coming Soon
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+You have to use this tools in your local machine.
+
+- NPM
+- NODEJS
+- GIT & GITHUB
+- Any Code Editor (VS Code... etc)
+
+### Setup
+
+Clone this repository to your desired folder:
+Example commands:
+
+```sh
+  cd my-folder
+  git init # if not initialized git on your folder
+  git clone git@github.com:hamzaw9/metrics.git
+  cd metrics
+  npm install # to install dependencies
+```
+
+### Run tests
+
+To run tests, run the following command on terminal:
+Example commands:
+
+Test for webhint errors
+
+```sh
+	npx hint .
+```
+
+Test for stylelint errors
+
+```sh
+  npx stylelint "**/*.{css,scss}"
+```
+
+Test for eslint errors
+
+```sh
+	npx eslint .
+```
+
+Trigger the build process of the project
+
+```sh
+	npm run build
+```
+
+Start the project
+
+```sh
+	npm run dev
+```
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Hamza Baig**
+
+- GitHub: [@hamzabaig](https://github.com/hamzaw9)
+- LinkedIn: [@hamzabaig](https://www.linkedin.com/in/hamzabaig-/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- UX improvements
+- Create additional pages
+- Desktop version
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+Your support is crucial to the success of this project! There are ways you can contribute and show your support:
+
+- Provide Feedback: If you've tried out the project or have any suggestions for improvement, we would love to hear from you. Your feedback is invaluable in shaping the future direction of the project.
+
+- Give a ⭐️ if you like this project!
+
+Remember, your support matters! Whether it's spreading the word, providing feedback, reporting issues, contributing code, or making a donation, every action makes a difference. Together, we can make this project even better.
+
+Thank you for your support and for being a part of this journey!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to give credit to list of contributors:
+
+- Nelson Sakwa Creator of design.
```

Comment:
- [ ] [Optional] Good job mentioning the author of the original design Nelson Sakwa, but it would be much better to add the link to her Behance account too.